lecture/views.py

```
@@ -45,19 +45,19 @@ def lecture_home(request: HttpRequest,slug:str) -> HttpResponse:
     # Get the video or return 404 if not found
     video = get_object_or_404(LectureVideo, id=video_id)
-    lecture=Lecture.objects.get(slug=slug)
+    module=Module.objects.get(id=module_id)
     # Ensure the user is enrolled in the related course
     if not request.user.is_staff and not video.lecture.course.enrollments.filter(student=request.user).exists():
         return HttpResponseForbidden("You are not enrolled in this course.")
 
     context = {
         "video": video,
-        "lecture":lecture
+        "module":module
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)